pkg/storage/tsdb/config.go

```diff
@@ -56,34 +56,17 @@ var (
 	errEmptyBlockranges             = errors.New("empty block ranges for TSDB")
 )
 
-// Config holds the config information for TSDB storage
+// Config holds the config information for the blocks storage.
 type Config struct {
```

Comment:
Do we want to rename this to `BlocksStorageConfig`? (Right now, the full name is `tsdb.Config`, which looks bit weird considering we also have `tsdb.TSDBConfig`. Perhaps we should rename the package instead. Not necessarily in this PR)

Reply:
Agree. Let's rename it.